spec/features/sign_up_spec.rb

```diff
@@ -0,0 +1,13 @@
+feature 'signing up' do
+  scenario 'user can sign up' do
+    sign_up
+
+    expect(page).to have_content 'Welcome to Chitter, MarcoAsensio!'
+  end
+
+  scenario 'same email cannot be used twice' do
```

Comment:
Does this rely on the other scenario being run first?  This is normally to be avoided.

Reply:
So, I imagine then the right way to test it would be to sign-up again (twice) with different data, right? Either that or to truncate the table in the test DB after running each test, right? Any preferences on better practices here? Thanks in advance!